crates/web5/src/http.rs

```diff
@@ -0,0 +1,138 @@
+use crate::errors::{Result, Web5Error};
```

Comment:
@diehuxx hold off on reviewing anything http-related until we have #354 merged, and I'll wrangle this around to use `http-std` as the default and enable developer-override